src/sensors/Particles.js

```diff
@@ -9,122 +10,103 @@
  * @param options - object with following keys:
  *
  *  * tfClient - the TF client handle to use
- *  * texture - (optional) Image url for a texture to use for the points. Defaults to a single white pixel.
- *  * rootObject (optional) - the root object to add this marker to
- *  * size (optional) - size to draw each point (default 0.05)
- *  * max_pts (optional) - number of points to draw (default 100)
```

Comment:
`texture` and `size` are gone here also. Could you add a check with deprecation warning?